uaclient/tests/test_cli.py

```diff
@@ -340,7 +340,9 @@ def test_function(args, cfg):
         assert mock.sentinel.success == ret
         lock_msg = "Operation in progress: some operation"
         assert [mock.call("", lock_msg)] == m_add_notice.call_args_list
-        assert [mock.call("", lock_msg)] == m_remove_notice.call_args_list
+        assert [
+            mock.call("", "Operation in progress.*")
+        ] == m_remove_notice.call_args_list
```

Comment:
should there be a test that we hit the new callable?

Reply:
Sure the `Operation in progress.*` was white box knowledge that the only place in uaclient.config.py which calls with that specific param is inside delete_cache_key("lock") but I'll make it more explicit for reviewers with a mock instead.